src/Compute/Compute.Test/ScenarioTests/ImageTests.ps1

```diff
@@ -342,123 +342,183 @@ function Test-DefaultImagesExistManual
         # UbuntuLTS test
         
         # Ubuntu2204 test
+        $vmname = "v1" + $rgname;
         $publisher = $images.Linux.Ubuntu2204.publisher;
```

Comment:
what is going on in this test?

Reply:
@grizzlytheodore 
This test 
1. Creates a unique VM name.
2. Grabs the image data from the json file in PS. 
3. Tries to Get that image to ensure it still exists. 
4. Uses that image to make a VM. 
5. If any errors occur, bad.